src/briefcase/ios.py

```diff
@@ -130,7 +130,7 @@ def set_device_target(self):
             data = json.loads(output)
 
             if self.os_version is None:
-                os_list = [label for label in data['devices'] if label.startswith('iOS')]
+                os_list = [label for label in data['devices'] if label.startswith('iOS') or label.startswith('com.apple.CoreSimulator.SimRuntime.iOS')]
```

Comment:
At column 120: (E501) line too long (151 > 119 characters)

Reply:
Yep, good idea. I will respect E501 too.

Reply:
To respect E501, I decomposed the one-liner. I hope that's OK.